### ISSUE TITLE
[BOUNTY] Port over Vore/chomp's vore system and related things ($75+)

### ISSUE THREAD

**Issue by BongaTheProto** [SPLURT-Station/S.P.L.U.R.T-Station-13#444]
## Contact:
Mossy#1471

## About the Bounty
What the title says. Our current vore system is rather similar to that of these two stations, so in some way this bounty would be about updating and adding the remaining features that we currently don't have to this system, and updating it to match their optimizations and mechanics wherever possible. 
Given that both vore's and chomp's systems have certain different mechanics and additions, the idea is taking vore's system (the most optimized one) and adding all of the extra things that chomp has added to it, liquid bellies for example.

There was a bounty on citadel that was basically this exact same thing but it was never completed. The idea of this one is bringing that bounty back to life. The code on it can probably be used as a base, and this bounty on itself also includes adding all of the features that were to be added on cit's ould bounty but were never finished alongside the rest of their system.

The bounty in question: https://github.com/Citadel-Station-13/Citadel-Station-13/pull/15388

If the pr is made directly to us, all I ask for is PLEASE make it as modular as possible, if that's even possible of course.

If you'd like to discuss the features and additions more in depth feel free to add me on discord! I can pay through paypal and you'll receive the payment once the code is merged into the main code.

**Comment by ArokyaMatthew** [comment]
@MosleyTheMalO  , i would like to contribute , is this still available?

**Comment by zhaog100** [comment]
I claim this bounty. GitHub: zhaog100 🌶️ (小米辣)
